### PR TITLE
Feature/code cleanup

### PR DESCRIPTION
**Pull request by vigneshgowrabathuni**
- Code cleanup changes

### CODE REVIEW

**Comment by gsundeep-tech** [comment]
@vigneshgowrabathuni main.css file is not required. I have tested it without the main.css and it works. when the react application is starting at runtime main.css file is being generated.

File: package.json [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
+    "start": "npm run build:css && react-scripts start",
+    "build": "npm run build:css && react-scripts build",
+    "test": "react-scripts test --watchAll",
+    "test:coverage": "npm run test --watchAll -- --coverage",
     "eject": "react-scripts eject",
     "format": "prettier --write src/**/*.{js,jsx}",
     "lint": "eslint --ext .jsx,.js ./src",
     "lint:fix": "npm run lint -- --fix",
-    "cssx": "postcss src/styles/tailwind.css -o src/styles/main.css"
+    "build:css": "postcss src/styles/tailwind.css -o src/styles/main.css"
```
Comment (gsundeep-tech) [inline package.json]: Somehow "build:css" was not working for me. so I have used cssx. Have you tested it?

File: public/index.html [hunk 2]
```diff
@@ -1,44 +1,21 @@
...
-    -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
+<head>
```
Comment (gsundeep-tech) [inline public/index.html:4]: Thanks for cleaning the html

File: package.json [hunk 3]
```diff
@@ -46,7 +46,6 @@
     ]
   },
   "dependencies": {
-    "@craco/craco": "^6.1.1",
```
Comment (gsundeep-tech) [inline package.json]: Thanks for removing the library. Is it working without this library?

File: src/App.css [hunk 4]
```diff
@@ -1,38 +0,0 @@
-.App {
-  text-align: center;
-}
-
```
Comment (gsundeep-tech) [inline src/App.css:4]: Thanks for cleaning the css

**Review by gsundeep-tech — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (vigneshgowrabathuni) [reply package.json]: @gsundeep-tech Yes it's working fine in local. `cssx `is not a good name so i have changes it to `build:css`

**Comment by gsundeep-tech** [comment]
> @vigneshgowrabathuni main.css file is not required. I have tested it without the main.css and it works. when the react application is starting at runtime main.css file is being generated.

@vigneshgowrabathuni  please remove the main.css file

File: package.json (hunk 3 above)
Comment (vigneshgowrabathuni) [reply package.json]: Yes It's working.

**Comment by vigneshgowrabathuni** [comment]
@gsundeep-tech I have removed the `main.css` file and added it in the` .gitignore` to skip in the future while pushing the code.

**Review by gsundeep-tech — APPROVED** [review]

**Pull request merged**